perf/stability/setup_tests.sh

```diff
@@ -30,6 +30,7 @@ function setup_tests() {
       "allconfig") setup_all_config;;
       "gateway-bouncer") ${WD}/gateway-bouncer/setup.sh;;
       "sds-certmanager") ${WD}/sds-certmanager/setup.sh;;
+      "mysql") ${WD}/mysql/setup.sh;;
```

Comment:
This shouldn't need a custom setup if we do the change above

Reply:
Still need since we need two deployments.

BTW I thought about doing a for-loop in helm template, but complicates the helm tmpl decides to install 2 deployment instead in bash.